database/index.js

```diff
@@ -0,0 +1,79 @@
+const dotenv = require('dotenv').config();
+const mongoose = require('mongoose');
+const faker = require('faker');
+
+mongoose.connect('mongodb://victor:victor123@ds125385.mlab.com:25385/products', {useNewUrlParser: true});
```

Comment:
you may want to refactor this to use your config.js file instead of hard coding in your password and username, obviously these are not legit usernames and passwords but still.